src/twisted/trial/_dist/disttrial.py

```diff
@@ -141,113 +337,113 @@ def _driveWorker(self, worker, result, testCases, cooperate):
 
         @param testCases: The global list of tests to iterate.
 
-        @param cooperate: The cooperate function to use, to be customized in
-            tests.
-        @type cooperate: C{function}
```

Comment:
I could see no value brought by using `cooperator` in this code.  Sending jobs to the worker processes is asynchronous and already gives up execution control.